Makefile

```diff
@@ -5,6 +5,7 @@ SHELL := /bin/bash
 OPENSHIFT_VERSION ?= 4.12.1
 PODMAN_VERSION ?= 4.3.1
 OKD_VERSION ?= 4.11.0-0.okd-2022-11-05-030711
+MICROSHIFT_VERSION ?= 4.12.1-202301261525.p0.g3db9e81.assembly.4.12.1.el8
```

Comment:
Could this be just `4.12.1`? Or do we expect to need multiple bundle `subversions` for the same `4.12.1` version?

Reply:
@cfergeau it would be `4.12.5`  not the full NVR of the rpm.